### PR TITLE
Add css class to field wrapper to correct error spacing

### PR DESCRIPTION
**Pull request by gkillough**

### CODE REVIEW

File: src/main/css/field.scss [hunk 1]
```diff
@@ -29,7 +29,7 @@ small {
 
 .fieldError {
   color: #a94442;
-  padding: 4px 0;
+  padding: 4px 0px 0px;
```
Comment (gkillough) [inline src/main/css/field.scss]: https://www.w3schools.com/css/css_padding.asp#midcontentadcontainer

Comment (jamesrichard91) [inline src/main/css/field.scss]: Should we add the last '0px' to cover all of the padding?

Comment (gkillough) [reply src/main/css/field.scss]: If the padding property has three values:
- padding: 25px 50px 75px;
- top padding is 25px
- right and left paddings are 50px
- bottom padding is 75px

Comment (jamesrichard91) [reply src/main/css/field.scss]: Well then what you have is perfect

**Pull request merged**